posts/promql-guide.md

```diff
@@ -0,0 +1,159 @@
+# PromQL for Humans
+
+PromQL is a built in query-language made for Prometheus. Here at Timber [we've found Prometheus to be awesome](https://timber.io/blog/prometheus-the-good-the-bad-and-the-ugly/), but PromQL difficult to wrap our heads around. This is our attempt to change that.
+
+## Basics
+
+### Instant Vectors
+
+_Only Instant Vectors can be graphed._
+
+`http_requests_total`
+
+![](./images/promql-guide/http_requests_total.png)
+
+This gives us all the http requests, but we've got 2 issues.
+1. There are too many data points to decipher what's going on.
+2. You'll notice that `http_requests_total` only goes up, because it's a [counter](https://prometheus.io/docs/concepts/metric_types/#counter). These are common in Prometheus, but not useful to graph.
+
+I'll show you how to approach both.
+
+It's easy to filter by label.
+
+`http_requests_total{job="prometheus", code="200"}`
+
+`!=` - not equal
```

Comment:
If you reference this here, I would use it in an example or discard it. You mention other operations below where it is also included.